lib/content-services/src/lib/agent/services/agent.service.ts

```diff
@@ -36,39 +38,88 @@ export class AgentService {
         this._mocked = mocked;
     }
 
+    agentsList$ = this.agentsList.asObservable();
+
     constructor(private apiService: AlfrescoApiService) {}
 
+    /**
+     * Gets all agents from cache. If cache is empty, fetches agents from backend.
+     *
+     * @returns Agent object containing the agents.
```

Comment:
Earlier it was an object (AgentPaging) but now it's list, so that description should be something like: 
@returns AgentWithAvatar[] list containing agents.